pages/about/projects/wai-coop/index.md

```diff
@@ -34,74 +34,73 @@ This page provides information on the Web Accessibility Initiative - Communities
+:  Provided responses to frequently asked questions (FAQ) on implementing the EU Web Accessibility Directive (WAD).
 
-**[EM Report Tool](https://www.w3.org/WAI/eval/report-tool/)**
-:  Updated functionality, improved user interface, improved translation options and updates to include new WCAG 2.1 and 2.2 success criteria.
-EM Report Tool
+**[Web Content Accessibility Guidelines Evaluation Methodology (WCAG-EM) Report Tool](https://www.w3.org/WAI/eval/report-tool/)**
+:  Updated functionality, improved user interface, improved translation options, and updated to include new WCAG 2.1 and 2.2 success criteria.
 
-**[Easy Checks - A First Review of Web Accessibility](https://www.w3.org/WAI/test-evaluate/easy-checks/)**
-:  Updates to the resource to simplify the presentation of each check, reduce the reliance on third-party tools and improve the readability of the content. This aims to make it easier to be used as a learning resource for people new to accessibility and a set of tools to perform some basic accessibility checks.
+**[Easy Checks &mdash; A First Review of Web Accessibility](https://www.w3.org/WAI/test-evaluate/easy-checks/)**
+:  Updated the resource to simplify the presentation of each check, reduce the reliance on third-party tools, and improve the readability of the content. The aim was to make it easier to use as a learning resource for people new to accessibility, and to create a set of tools to perform some basic accessibility checks.
 
 ## Participation
 {:#participate}
 
-Most of the project work listed above was developed through or in coordination with:
+Most of the project work listed above was developed through or in coordination with the:
 
 -   [Accessibility Education and Outreach Working Group (EOWG)](/about/groups/eowg/)
 -   [Accessibility Conformance Testing (ACT) Task Force](/about/groups/task-forces/conformance-testing/)
 -   [Research Questions Task Force (RQTF)](/about/groups/task-forces/research-questions/)
 
-See also [Participating in WAI](http://www.w3.org/WAI/participation). If you have any questions, contact [Kevin White](https://www.w3.org/staff/#kevin).
-
 ## About the Project
 {:#about}
 
-WAI-CooP, Communities of Practice, is a Coordination and Support Action project, co-funded by the European Commission (EC) under the Horizon 2020 program (Grant Agreement 101004794). The project is schedule to start on 1 January 2021 for a duration of three years. WAI-CooP support implementation of the international standards for digital accessibility through:
+WAI-CooP, Communities of Practice, was a Coordination and Support Action project, co-funded by the European Commission (EC) under the Horizon 2020 program (Grant Agreement 101004794). The project started on 1 January 2021 and was completed in December 2023. WAI-CooP supported implementation of the international standards for digital accessibility through:
```

Comment:
I'm pretty sure this is correct. Will publish with it and we can correct later if needed.